starter-code/pizza.js

```diff
@@ -1 +1,27 @@
-// Write your Pizza Builder JavaScript in this file.
+$(".sauce").removeClass("sauce-white");
+$(".crust").removeClass("crust-gluten-free");
+
+$(".btn-green-peppers").click(function() {
+  $(".green-pepper").toggle();
+  $(".btn-green-peppers ").toggleClass("active");
+});
+
+$(".btn-mushrooms").click(function() {
+  $(".mushroom").toggle();
+  $(".btn-mushrooms").toggleClass("active");
+});
+
+$(".btn-pepperonni").click(function() {
+  $(".pep").toggle();
+  $(".btn-pepperonni").toggleClass("active");
+});
+
+$(".btn-crust").click(function() {
+  $(".crust").toggleClass("crust-gluten-free");
+  $(".btn-crust").toggleClass("active");
+});
+
+$(".btn-sauce").click(function() {
+  $(".sauce").toggleClass("sauce-white");
+  $(".btn-sauce").toggleClass("active");
+});
```

Comment:
It was a good beginning. Here is a solution that includes all the features (correct the fact that the buttons sauce and crust are the opposite of what they should be): 
```
// Iteration 1

$(".btn-green-peppers").click(function() {
  $(".green-pepper").toggle();
  updateButtonStatus();
});

$(".btn-mushrooms").click(function() {
  $(".mushroom").toggle();
  updateButtonStatus();
});

$(".btn-pepperonni").click(function() {
  $(".pep").toggle();
  updateButtonStatus();
});


// Iteration 2

$(".btn-sauce").click(function() {
  var isWhiteSauce = $(".sauce").hasClass("sauce-white");
  if (isWhiteSauce) {
    $(".sauce").removeClass("sauce-white");
  } else {
    $(".sauce").addClass("sauce-white");
  }
  updateButtonStatus();
});

$(".btn-crust").click(function() {
  var isGlutenFree = $(".crust").hasClass("crust-gluten-free");
  if (isGlutenFree) {
    $(".crust").removeClass("crust-gluten-free");
  } else {
    $(".crust").addClass("crust-gluten-free");
  }
  updateButtonStatus();
});

// Iteration 3

function updateButtonStatus() {
  var isGreenPepper = $(".green-pepper").is(":visible");
  var isMushroom = $(".mushroom").is(":visible");
  var isPepperonni = $(".pep").is(":visible");
  var isWhiteSauce = $(".sauce").hasClass("sauce-white");
  var isGlutenFree = $(".crust").hasClass("crust-gluten-free");

  if (isGreenPepper) {
    $(".btn-green-peppers").addClass("active");
  } else {
    $(".btn-green-peppers").removeClass("active");
  }

  if (isMushroom) {
    $(".btn-mushrooms").addClass("active");
  } else {
    $(".btn-mushrooms").removeClass("active");
  }

  if (isPepperonni) {
    $(".btn-pepperonni").addClass("active");
  } else {
    $(".btn-pepperonni").removeClass("active");
  }

  if (isWhiteSauce) {
    $(".btn-sauce").addClass("active");
  } else {
    $(".btn-sauce").removeClass("active");
  }

  if (isGlutenFree) {
    $(".btn-crust").addClass("active");
  } else {
    $(".btn-crust").removeClass("active");
  }
}
```